.github/ISSUE_TEMPLATE/4_change_request.yml

```diff
@@ -0,0 +1,16 @@
+name: 🤌 Complain
```

Comment:
This template, like the discussion template, should just offer a single field. The advantage of forms would be that we can provide informational text and links around the field without leaving HTML comments on the field itself.

Reply:
I made it a single textarea,

Reply:
Or, maybe it should have a `input` i.e. a single-line field first, where the user just enters the feature he doesn't like? 
And, then use a `textarea` to explain their reasoning?